docs/source/guide/deployment/planning/volumes.rst

```diff
@@ -0,0 +1,68 @@
+#######
+Volumes
+#######
+
+As the Repository Service for TUF provides containers images, it is required to
+work with volumes for persistent data.
```

Comment:
Is this multiple containers?  I believe in this instance we can still use a single term of "container".

Reply:
We have two container images: rstuf-worker and rstuf-api